connectors/src/api_server.ts

```diff
@@ -27,6 +27,14 @@ import logger from "@connectors/logger/logger";
 import { authMiddleware } from "@connectors/middleware/auth";
 
 export function startServer(port: number) {
+  process.on("unhandledRejection", (reason, promise) => {
+    logger.error("Unhandled Rejection at:", promise, "reason:", reason);
```

Comment:
are you sure this logger syntax is correct? Isn't it logger.error({k:v}, message)?

Reply:
The syntax is correct, but the format you suggested is a lot better...

Reply:
Yep let's use an object in a follow-up PR :+1: